service/routers/decks.js

```diff
@@ -0,0 +1,184 @@
+import { Router } from 'express'
+import { body } from 'express-validator'
+import { User } from '../models/User.js'
+
+const decksRouter = Router()
+
+const getDecks = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser" || req.user.userId == req.params.id) {
+    const { userId, role } = req.user
+    console.log(`Other data from the token ${role}`)
+    try {
+      const user = await User.findById(req.params.id)
```

Comment:
What will this actually return?

Reply:
In req.params.userid, when a userId is passed by the user via the API all the decks of the user, will be returned.